lib/bes/buildbuddy.go

```diff
@@ -20,6 +20,8 @@ var (
 	getInvocationEndpoint = mustParseURL("rpc/BuildBuddyService/GetInvocation")
 )
 
+var _ httpDoer = http.DefaultClient
+
```

Comment:
Why is this needed?

Reply:
When duck typing a struct for testing usually it's polite to name where it came from.